EleksTubeHAX_pio/src/_USER_DEFINES - empty.h

```diff
@@ -49,12 +61,10 @@
 
   // TODO: Store the dimming values and dimming times in the NVS partition to keep the last dimming value and not use the hard coded values
   // make the times and values adjustable in the menu and/or via MQTT for both main and backlight dimming
-#endif
 
-#ifdef DIM_WITH_ENABLE_PIN_PWM
-  // skip reinitialization if using PWM to control backlight since actual TFTs are never deactivated
-  // disable by commenting the following line if this causes wake up issues from disabled (brightness 0) state
-  #define TFT_SKIP_REINIT
+  // TODO: Save the values changed via MQTT/in HA in the NVS partition to keep the values after a reboot. Maybe define a "save command" in HA or trigger after 
+  // a few minutes of inactivity only if changed something or in the "free time" of the loop cycle...
+  // Save it every time receiving MQTT commands is a BAD idea, we know that already ;)
```

Comment:
I am quite interested why that's a bad idea? :D

Reply:
The frequency of the MQTT messages are high and if we safe each time after a change, we are writing to the flash (NVS) and that takes some time...In the past it was done all the time, a message arrived and it made the clock behave strange and unpredicatable. So the line with the safe command was commented out. 
It is generally not a good idea, to "penetrate" the flash like that...limited write cycles :)

Reply:
I just saw, that it is already implemented...60 secs after the last received MQTT topic, the save method is called...